tests/helpers/perf_run_helper.rb

```diff
@@ -786,6 +801,9 @@ def get_perf_result
       # TODO: tar archive file before copying to avoid timeouts with soak results
```

Comment:
Since you're making an update here would you mind removing this TODO that I previously added now that SLV-560 is done?

Reply:
You bet. Done.

> Since you're making an update here would you mind removing this TODO that I previously added now that SLV-560 is done?

You bet. Done.

Reply:
Thanks!